boards/riscv/niosv_g/niosv_g.dts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2023, Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <niosv/niosv-g.dtsi>
+
+/ {
+	model = "niosv_g";
+	compatible = "intel,niosv_g";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,sram = &sram0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <50000000>;
+};
+
+&sram0 {
+	reg = <0x0 0x40000>;
+};
+
+&mtimer {
+	reg = <0x90000 0x10>;
+};
```

Comment:
is reg board-specific?

Reply:
As this is a FPGA design example, user has flexibility to change the design which include this address per their requirement. These nodes are populated on board dts file to ease user to change the address accordingly so that generated Zephyr executable should work accordingly if they using different address in their FPGA design.